pkg/controller/containersource/reconcile_test.go

```diff
@@ -94,7 +94,7 @@ var testCases = []controllertesting.TestCase{
 		},
 		ReconcileKey: fmt.Sprintf("%s/%s", testNS, containerSourceName),
 		Scheme:       scheme.Scheme,
-		WantErrMsg:   "sink does not contain address",
+		WantErrMsg:   "sink \"testnamespace/testunaddressable\" (duck.knative.dev/v1alpha1.KResource) does not contain address",
```

Comment:
I suggest changing the surrounding quotes to backticks and unescaping the inner quotes. Mostly for consistency with line 148, but also the backticks read cleaner to me.

Optionally, remove the inner quotes entirely since spaces aren't allowed in object names.